main/autocode/src/org/ejml/GenerateJavaCode32.java

```diff
@@ -54,6 +54,8 @@ public GenerateJavaCode32() {
         prefix32.add("FUnary");
         prefix64.add("DBinary");
         prefix32.add("FBinary");
+        prefix32.add("IFBinary");
```

Comment:
So close...

Reply:
Oh wow :sweat_smile: 
Sorted by order ... Now it must be correct!
Maybe introducing a Class-Annotation like `@PrimitiveSpecific(floatName="FBinary")` would be an idea as this list will continue to grow.
Based on the annotated classes, you could fill part of the `prefix32`, `prefix64` and `converter.replacePattern`.

Reply:
That's not a bad idea. The entire project can be scanned fairly quickly for that annotation. Although it might not be an annotation in the end but a comment. The auto code generator isn't part of javac and requiring an added dependency for the annotation, which is never really used, would be annoying.

Reply:
I will create a ticket and pick it up when I find some spare time. sounds interesting to implement.

Reply:
Let me know when you get serious. I might have the source parser already implemented here for lint checks https://github.com/lessthanoptimal/BoofCV/tree/SNAPSHOT/main/checks/src/main/java/boofcv . I'm planning on unleashing some of those checks on EJML after I clean the code up a bit and maybe move it into its own project with a few other tools. Although it might be better to see if Error Prone is a better location for these types of checks.

Reply:
Ok, I will let you know. Created #105 to keep track of it :) 

Some checks should be definitely doable via Error Prone.